logging/appender.go

```diff
@@ -9,6 +9,9 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+// DefaultTimeFormatStr is the default time format string for log appenders.
+var DefaultTimeFormatStr = "2006-01-02T15:04:05.000Z0700"
```

Comment:
[q] is the meant to editable by users? if so I'd prefer to rename this to just `TimeFormat`. 

otherwise:

```suggestion
const DefaultTimeFormatStr = "2006-01-02T15:04:05.000Z0700"
```

Reply:
It is not meant to be editable; good catch! I'll remove the `Default` language.

Reply:
My bad 😮‍💨 , missed that you were suggesting `const` and a potential rename. I'll keep `DefaultTimeFormatStr` and make it `const`.

Reply:
no worries! i said it in a confusing way. the main thing is making it a `const` if it's not meant to editable - either name is reasonable

Reply:
> is there any way for users to set a different date format currently?

There is no way currently! Could file a ticket if that seems like a useful feature.

Reply:
cool, good to know! no need to make a ticket since it doesn't seem like anyone has been asking for it. I was just curious for my own understanding